src/components/SearchResults.jsx

```diff
@@ -0,0 +1,32 @@
+import { Box } from '@chakra-ui/react';
+import React from 'react';
+import { useQuery } from 'react-query';
+
+import CardList from './CardList';
+
+function SearchResults({ queryURL }) {
+  const { isLoading, error, data } = useQuery(
+    ['foodData', queryURL],
+    async () =>
+      fetch(queryURL).then((response) => {
+        if (response.ok) {
+          return response.json();
+        }
+        throw new Error(error.message);
```

Comment:
It now fails more gracefully when a request is denied server-side. After I moved API keys to the .env, Vercel no longer had access to them, @tonoezep u'll have to add them to the deployment bot.
I'm pasting them here, because there's no real risk with these keys.
Fetch considers any response as successful, even denial errors, so I had to force it to consider anything other than OK as an error, [it's a documented issue with Fetch](https://react-query.tanstack.com/guides/query-functions#usage-with-fetch-and-others-clients-that-do-not-throw-by-default). I need to find a way to make react-query understand the error message so it can be rendered, but I don't know how.
Either that or use axios instead of Fetch, axios works out of the box, it takes care of handling server errors and such, in order to give the adequate inputs to react-query.

REACT_APP_QUICKSEARCH_KEY=2fc9bd5f970a4db7a0d322af9781240d
REACT_APP_ADVANCEDSEARCH_KEY=2fc9bd5f970a4db7a0d322af9781240d
 find a way to do it.